aireview/src/aireview/config.py

```diff
@@ -1,11 +1,11 @@
 from pydantic_settings import BaseSettings
 
-class Config(BaseSettings):
+class AIReviewSettings(BaseSettings):
     dust_api_key: str
```

Comment:
**naming**: Nested class has the same name as the parent class 'AIReviewSettings'. This creates confusion and potential naming conflicts. Consider renaming the inner class to 'Config' as per pydantic's convention.